src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/Machine.cs

```diff
@@ -129,5 +129,10 @@ public enum Machine : ushort
         /// ARM64
         /// </summary>
         Arm64 = 0xAA64,
+
+        /// <summary>
+        /// LOONGARCH64
+        /// </summary>
+        Loongarch64 = 0xDD64,
```

Comment:
The values in this enum are only relevant on Windows. I think this change should be reverted.

Notice that wasm or s390x are not in this enum either since Windows OS does not run on these architectures.

Reply:
I don't think so, see https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#machine-types
`IMAGE_FILE_MACHINE_LOONGARCH64` is already a valid PE header value.

Reply:
Ok, if the PE spec defines the value, it should be fine to add it here. The value needs to match what is in the PE spec.

Reply:
Thanks, I will update it as the spec.